server/src/auth-router.ts

```diff
@@ -76,8 +76,8 @@ if (DOMAIN) {
       clientSecret: CLIENT_SECRET,
       callbackURL:
         (({
-          stage: 'https://voice.allizom.org',
-          prod: 'https://voice.mozilla.org',
+          stage: 'https://commonvoice.allizom.org',
+          prod: 'https://commonvoice.mozilla.org',
```

Comment:
we haven't modified the callback in Auth0 side, is the app enforcing somehow this callback?